src/docs/doxygen/GeosxConfig.hpp

```diff
@@ -69,11 +69,11 @@
 #define GEOSX_USE_PETSC
 
 /// Choice of global linear algebra interface (CMake option GEOSX_LA_INTERFACE)
-#define GEOSX_LA_INTERFACE Trilinos
+#define GEOSX_LA_INTERFACE Hypre
 /// Macro defined when Trilinos interface is selected
-#define GEOSX_LA_INTERFACE_TRILINOS
+/* #undef GEOSX_LA_INTERFACE_TRILINOS */
 /// Macro defined when Hypre interface is selected
-/* #undef GEOSX_LA_INTERFACE_HYPRE */
+#define GEOSX_LA_INTERFACE_HYPRE
 /// Macro defined when PETSc interface is selected
```

Comment:
I do not think we want to commit that to the main branch

Reply:
Please merge the Pangea build also, because each time I checkout from develop I need to redo all the modifications for allowing the Pangea build.

Reply:
Please to not commit this generated file.
I do not think it should be generated anyhow, but so far it's like that.